.ci/configuration_aarch64.yaml

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2018 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# test components
+test:
+  - functional
+  - docker
+
+# test units for docker integration
+docker:
+  Describe:
+    - docker create
```

Comment:
Not knowing much about ginkgo, I don't really understand how this is working - could you explain?

Reply:
Also, if this config file is skipping tests, we need a github URL for each skipped test (or test category) with details of why the test(s) is/are being skipped.

Reply:
What docker.Describe,  docker.Context and docker.It offer is part of the values of ${FOCUS} in regex format. 
Copying from ginkgo documentation:
-focus string
    	If set, ginkgo will only run specs that match this regular expression.
-skip string
    	If set, ginkgo will only run specs that do not match this regular expression.
We only offer '-focus' flag, maybe we should also offer '-skip'.
ginkgo will search 'Describe', 'Context', and 'It' to match this regular expression.
Describe("context need to be matched", func() { ... })
Context("context need to be matched", func() { ... })
It("context need to be matched", func() { ... })

Reply:
For now, configuration_aarch64.yaml is just a demo for displaying how filter scheme works.
specific values are not defined yet.
we are working on verify test cases and integration units in arm arch, but still a long list to do.
in the future, we will add more and more bits into the successful lists(configuration_aarch64.yam).
just like what kaly said in #516.